kube/overlays/dev-integration-test/parallelize-worker.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airbyte-worker
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      airbyte: worker
+  template:
+    metadata:
+      labels:
+        airbyte: worker
```

Comment:
why do we need these lines. isn't the only thing we need in the overlap `replicas: 2`? could just be a gap in my kustomization undersstanding.

Reply:
I think you're right. I'll try cutting out most of the `spec`